library/std/src/sys_common/mod.rs

```diff
@@ -20,6 +20,8 @@
 #[cfg(test)]
 mod tests;
 
+#[cfg(target_os = "windows")]
+pub mod args;
```

Comment:
I think `wtf8_args` is better name.

Reply:
It doesn't handle just any WTF-8 encoded argument list strings, but only those that are used on Windows (and UEFI). It would fail to parse an argument lists string as you would enter in bash, zsh, fish, ...